### PR TITLE
PRLC100-153 safety_concerns abduction passport-amount changes

### PR DESCRIPTION
**Pull request by kumarmadurai**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/steps/c100-rebuild/safety-concerns/abduction/passport-amount/content.ts [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+  childrenMoreThanOnePassport: 'Do the children have more than one passport?',
+  possessionChildrenPassport: 'Who is in possession of the children’s passports?',
+  select_all_relevant: 'Select all that apply',
+  one: 'Yes',
+  two: 'No',
+  option1: 'Mother',
+  option2: 'Father',
+  option3: 'Other',
+  otherDetails: 'Provide more details',
+  guidanceParentalChildHyperlink: 'https://www.gov.uk/government/collections/child-abduction',
```
Comment (channa-patil) [inline src/main/steps/c100-rebuild/safety-concerns/abduction/passport-amount/content.ts:19]: @kumarmadurai We have created a common content.ts & template.njk files for these links.
Please remove these links.

File: src/main/steps/c100-rebuild/safety-concerns/abduction/passport-amount/template.njk [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{% extends "common/template.njk" %}
+
+
+{% block page_content %}
+<span class="govuk-caption-xl">{{caption}}</span>
+<h1 class="govuk-heading-xl govuk-!-margin-bottom-7">{{title}}</h1>
+<p class="govuk-body-m">{{paragraph1}}</p>
+<p class="govuk-body-m">{{paragraph2}}</p>
+{% endblock %}
+{% block page_safetyConcerns %}
```
Comment (channa-patil) [inline src/main/steps/c100-rebuild/safety-concerns/abduction/passport-amount/template.njk:10]: @kumarmadurai Please include common template.njk from abduction & remove these links added here.

**Review by channa-patil — CHANGES_REQUESTED** [review]

File: src/main/steps/c100-rebuild/safety-concerns/abduction/passport-amount/template.njk (hunk 2 above)
Comment (kumarmadurai) [reply src/main/steps/c100-rebuild/safety-concerns/abduction/passport-amount/template.njk:10]: fixed

File: src/main/steps/c100-rebuild/safety-concerns/abduction/passport-amount/content.ts (hunk 1 above)
Comment (kumarmadurai) [reply src/main/steps/c100-rebuild/safety-concerns/abduction/passport-amount/content.ts]: fixed

File: src/main/steps/c100-rebuild/safety-concerns/abduction/template.njk [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
-<a href="{{getHelpOnReturningChildAbroadLink}}" id="getHelpOnReturningChildAbroad" target="_blank" class="govuk-link">{{getHelpOnReturningChildAbroadLabel}}</a>
-</br>
-<a href="{{preventChildFromObtainingPassportLink}}" id="preventChildFromObtainingPassport" target="_blank" class="govuk-link">{{preventChildFromObtainingPassportLabel}}</a>
-</p>
\ No newline at end of file
+<a href="{{guidanceParentalChildHyperlink}}" id="guidanceParentalChildLabel" target="_blank" class="govuk-link">{{ guidanceParentalChildLabel }}</a>
+<br>
+<a href="{{helpForAbroadChildHyperlink}}" id="helpForAbroadChildLabel" target="_blank" class="govuk-link">{{ helpForAbroadChildLabel }}</a>
+<br>
+<a href="{{stopChildGettingPassportHyperlink}}" id="idestopChildGettingPassportLabelntifySignsOfDomesticAbuseLabel" target="_blank" class="govuk-link">{{ stopChildGettingPassportLabel }}</a>
```
Comment (channa-patil) [inline src/main/steps/c100-rebuild/safety-concerns/abduction/template.njk:6]: @kumarmadurai id field got some junk value here by mistake I think. Can we please correct ?

File: src/main/steps/c100-rebuild/safety-concerns/abduction/content.ts [hunk 4]
```diff
@@ -1,21 +1,21 @@
 import { TranslationFn } from '../../../../app/controller/GetController';
 
 const en = () => ({
-  guidanceOnAbductionLabel: 'Guidance on parental child abduction',
-  guidanceOnAbductionLink: 'https://www.gov.uk/government/collections/child-abduction',
-  getHelpOnReturningChildAbroadLabel: 'Get help to return a child from abroad or arrange contact',
-  getHelponReturningChildAbroadLink: 'https://www.gov.uk/return-or-contact-abducted-child',
-  preventChildFromObtainingPassportLabel: 'Stop a child from getting a passport',
-  preventChildFromObtainingPassportLink: 'https://www.gov.uk/stop-child-passport',
+  guidanceParentalChildHyperlink: 'https://www.gov.uk/government/collections/child-abduction',
```
Comment (channa-patil) [inline src/main/steps/c100-rebuild/safety-concerns/abduction/content.ts:4]: Did you rename al these label & link field names ?

**Review by channa-patil — CHANGES_REQUESTED** [review]

File: src/main/steps/c100-rebuild/safety-concerns/abduction/template.njk (hunk 3 above)
Comment (kumarmadurai) [reply src/main/steps/c100-rebuild/safety-concerns/abduction/template.njk]: Fixed

File: src/main/steps/c100-rebuild/safety-concerns/abduction/content.ts (hunk 4 above)
Comment (kumarmadurai) [reply src/main/steps/c100-rebuild/safety-concerns/abduction/content.ts]: fixed

**Pull request merged**